taipy/core/data/_file_datanode_mixin.py

```diff
@@ -77,8 +77,9 @@ def path(self) -> Any:
     @path.setter
     def path(self, value):
         self._path = value
-        self.properties[self._PATH_KEY] = value
-        self.properties[self._IS_GENERATED_KEY] = False
+        properties = self.properties
+        properties[self._PATH_KEY] = value
+        properties[self._IS_GENERATED_KEY] = False
```

Comment:
I don't see the purpose for this change.

Reply:
to reduce the amount of time we need to call `self.properties` as each time it's called, we are pulling from the repo again

Reply:
ah I see what you meant! let me remove this change!